LootLockerServerSDK/Source/LootLockerServerSDK/Public/LootLockerServerForBlueprints.h

```diff
@@ -937,4 +939,177 @@ class LOOTLOCKERSERVERSDK_API ULootLockerServerForBlueprints : public UObject
+     *
+     * @param PlayerID ID of the player for whom to subtract points from the specified progression
+     * @param ProgressionKey Key of the progression to subtract points from for the specified player
+     * @param Amount The number of points to subtract from the specified progression (only positive integers allowed)
+     * @param OnCompletedRequest Delegate for handling the server response
+     */
+    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Player Progressions")
+    static void SubtractPointsFromProgressionForPlayer(int PlayerID, const FString& ProgressionKey, int32 Amount, const FLootLockerServerSinglePlayerProgressionWithRewardsResponseBP& OnCompletedRequest);
+
+    /**
+     * Reset the specified progression for the specified player
+     * https://ref.lootlocker.com/server-api/#reset-a-player-progression
+     *
+     * @param PlayerID ID of the player for whom to reset the specified progression
+     * @param ProgressionKey Key of the progression to reset for the specified player
+     * @param OnCompletedRequest Delegate for handling the server response
+     */
+    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Player Progressions")
+    static void ResetProgressionForPlayer(int PlayerID, const FString& ProgressionKey, const FLootLockerServerSinglePlayerProgressionWithRewardsResponseBP& OnCompletedRequest);
+
+    /**
+     * Delete the specified progression for the specified player
+     * https://ref.lootlocker.com/server-api/#delete-a-player-progression
+     *
+     * @param PlayerID ID of the player for whom to delete the specified progression
+     * @param ProgressionKey Key of the progression to delete for the specified player
+     * @param OnCompletedRequest Delegate for handling the server response
+     */
+    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Player Progressions")
+    static void DeleteProgressionForPlayer(int PlayerID, const FString& ProgressionKey, const FLootLockerServerDeletePlayerProgressionResponseBP& OnCompletedRequest);
+
+    //==================================================
+    // Character Progressions https://ref.lootlocker.com/server-api/#character-progressions
+    //==================================================
+
+    /**
+     * Get a list of progressions for the specified player and character
+     * https://ref.lootlocker.com/server-api/#get-character-progressions
+     *
+     * @param PlayerID ID of the player the specified character belongs to
+     * @param CharacterID ID of the character for which to get progressions
+     * @param OnCompletedRequest Delegate for handling the server response
+     */
+    UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Character Progressions")
+    static void GetProgressionsForCharacter(int PlayerID, int CharacterID, const FLootLockerServerCharacterProgressionListResponseBP& OnCompletedRequest);
```

Comment:
Same thing here as with the Player Progression, add something about the amount being returned.